tasks/yesno.py

```diff
@@ -0,0 +1,301 @@
+
```

Comment:
load_hypev is unusable because the training only works with pairs <question>-<sentence> and not with <question>-<bunch of sentences>. The precision hypev shows is only averaged over questions after the training. 
The correct averaging baseline should use the same end-to-end approach as the current clasrel implementation only with simple average at the end.